src/sys/petsclib.cxx

```diff
@@ -13,7 +13,7 @@ int *PetscLib::pargc = nullptr;
 char ***PetscLib::pargv = nullptr;
 PetscLogEvent PetscLib::USER_EVENT = 0;
 
-PetscLib::PetscLib() {
+PetscLib::PetscLib(Options* opt) : options_prefix("") {
```

Comment:
The default `string` constructor is the empty string, so we can either drop `options_prefix` from the initaliser list, or replace it with L38 perhaps?

Reply:
:+1: can drop it. Is it worth putting `= ""` in the header to be explicit instead of relying on the default constructor?

Reply:
No, the default constructor is in the standard, so it's already explicit enough.